app/views/coc/finished.scala.html

```diff
@@ -38,5 +38,9 @@ <h1 class="heading-large"><img src="@routes.Assets.at("images/tick.png")" alt=""
 <p>@Html(Messages("pages.coc.finish.junk"))</p>
 <h2 class="heading-medium">@Html(Messages("pages.coc.finish.whn"))</h2>
 <p>@Html(Messages("pages.coc.finish.para1"))</p>
-<p>@Html(Messages("pages.coc.finish.para2"))</p>
+<p>
```

Comment:
Does the spacing render properly for the split messages? I feel like there will be no space between each message, but unsure.

Reply:
<img width="620" alt="Screenshot 2020-04-09 at 09 53 31" src="https://user-images.githubusercontent.com/33066342/78877039-25e1a400-7a48-11ea-88ee-be8f1ba3ee15.png">

This seems to render as expected